src/keen-slider.js

```diff
@@ -159,30 +159,38 @@ function KeenSlider(initialContainer, initialOptions = {}) {
   }
 
   function eventsAdd() {
+    const isEventPassive = !(!isTouchable() || !touchActive)
+
     eventAdd(window, 'orientationchange', sliderResizeFix)
     eventAdd(window, 'resize', () => sliderResize())
     eventAdd(container, 'dragstart', function (e) {
       if (!isTouchable()) return
       e.preventDefault()
```

Comment:
What is the likelihood of `isTouchable` returning a different value after `eventsAdd` is executed?

If it's not likely to change, I'd add the passive property onto `dragstart` too